src/main/scala/no/ndla/articleimport/ArticleImportProperties.scala

```diff
@@ -42,7 +42,7 @@ object ArticleImportProperties extends LazyLogging {
   val nodeTypeH5P: String = "h5p_content"
   val supportedContentTypes = Set("fagstoff", "oppgave", "veiledning", "aktualitet", "emneartikkel", nodeTypeBegrep, nodeTypeVideo, nodeTypeH5P)
 
-  val oldCreatorTypes = List("opphavsmann", "fotograf", "kunstner", "redaksjonelt", "forfatter", "manusforfatter", "innleser", "oversetter", "regissør", "illustratør", "medforfatter", "komponist")
+  val oldCreatorTypes = List("opphavsmann", "fotograf", "kunstner", "forfatter", "manusforfatter", "innleser", "oversetter", "regissør", "illustratør", "medforfatter", "komponist")
   val creatorTypes = List("originator", "photographer", "artist", "editorial", "writer", "scriptwriter", "reader", "translator", "director", "illustrator", "cowriter", "composer")
```

Comment:
Må vi ikke også fjerne `editorial` fra `creatorTypes` her? 
Vil det ikke bli en mismatch når den her blir zippa sammen? Kanskje det hadde vært smartere å teste om det var noen grunn til at jeg brukte lister istedet for map?